BibBuddy/app/src/main/java/de/bibbuddy/AsDefaultAppFragment.java

```diff
@@ -0,0 +1,500 @@
+    mainActivity.setVisibilitySortButton(true);
+
+    sortBtn.setOnClickListener(v -> handleSortShelf());
+  }
+
+  @Override
+  public void onCreateOptionsMenu(@NonNull Menu menu, MenuInflater inflater) {
+    inflater.inflate(R.menu.fragment_library_menu, menu);
+    super.onCreateOptionsMenu(menu, inflater);
+  }
+
+  @SuppressLint("NonConstantResourceId")
+  @Override
+  public boolean onOptionsItemSelected(MenuItem item) {
+    switch (item.getItemId()) {
+      case R.id.menu_rename_shelf:
+        handleRenameShelf();
+        break;
+
+      case R.id.menu_delete_shelf:
+        handleDeleteShelf();
+        break;
+
+      case R.id.menu_help_library:
+        handleManualLibrary();
+        break;
+
+      case R.id.menu_imprint:
+        ((MainActivity) requireActivity()).openImprint();
+        break;
+
+      default:
+        break;
+    }
+
+    return super.onOptionsItemSelected(item);
+  }
+
+  private void handleManualLibrary() {
+    HelpFragment helpFragment = new HelpFragment();
+    String htmlAsString = getString(R.string.library_help_text);
+
+    Bundle bundle = new Bundle();
+    bundle.putString(LibraryKeys.MANUAL_TEXT, htmlAsString);
+    helpFragment.setArguments(bundle);
+
+    helpFragment
+        .show(requireActivity().getSupportFragmentManager(), LibraryKeys.FRAGMENT_HELP_VIEW);
+  }
+
+  @Override
+  public void onPrepareOptionsMenu(Menu menu) {
+    MenuItem renameShelf = menu.findItem(R.id.menu_rename_shelf);
+    MenuItem deleteShelf = menu.findItem(R.id.menu_delete_shelf);
+
+    if (selectedShelfItems == null || selectedShelfItems.isEmpty()) {
+      renameShelf.setVisible(false);
+      deleteShelf.setVisible(false);
+
+    } else if (selectedShelfItems.size() != 1) {
+      renameShelf.setVisible(false);
+      deleteShelf.setVisible(true);
+
+    } else {
+      renameShelf.setVisible(true);
+      deleteShelf.setVisible(true);
+
+    }
+
+  }
+
+  private void handleDeleteShelf() {
+    AlertDialog.Builder alertDeleteShelf = new AlertDialog.Builder(context);
+    alertDeleteShelf.setCancelable(false);
+    setAlertMessage(alertDeleteShelf);
+
+    if (selectedShelfItems.size() > 1) {
+      alertDeleteShelf.setTitle(R.string.delete_shelves);
+    } else {
+      alertDeleteShelf.setTitle(R.string.delete_shelf);
+    }
+
+    alertDeleteShelf.setNegativeButton(R.string.cancel, (dialog, which) -> deselectLibraryItems());
+
+    alertDeleteShelf.setPositiveButton(R.string.delete, (dialog, which) -> {
+      performDeleteShelf();
+      deselectLibraryItems();
+    });
+
+    alertDeleteShelf.show();
+  }
+
+  private void setAlertMessage(AlertDialog.Builder alertDeleteShelf) {
+    alertDeleteShelf.setMessage(
+        getString(R.string.delete_shelf_message)
+            + convertShelfListToString(selectedShelfItems)
+            + getString(R.string.delete_counter_msg)
+            + getBooksToDeleteNumber(selectedShelfItems)
+            + " und " + getNotesToDeleteNumber(selectedShelfItems) + " "
```

Comment:
Instead of "und" it is better to get the text via getString(R.string.and) because "und" is German and for possible languages in the future it is better to do that.